wadas/domain/custom_classification_mode.py

```diff
@@ -0,0 +1,71 @@
+"""Custom Classification module."""
+
+import logging
+from queue import Empty
+
+from wadas.domain.animal_detection_mode import AnimalDetectionAndClassificationMode
+from wadas.domain.camera import img_queue
+
+logger = logging.getLogger(__name__)
+
+
+class CustomClassificationMode(AnimalDetectionAndClassificationMode):
+    """Custom Classification Mode class."""
+
+    def __init__(self, target_animal_label="bear"):
+        super().__init__()
+        self.process_queue = True
+        self.target_animal_label = target_animal_label
+
+    def run(self):
+        """WADAS custom classification mode"""
+
+        self._initialize_process()
+        self.check_for_termination_requests()
+
+        # Run detection model
+        while self.process_queue:
+            self.check_for_termination_requests()
+            # Get image from motion detection notification
+            # Timeout is set to 1 second to avoid blocking the thread
+            try:
+                cur_img = img_queue.get(timeout=1)
+            except Empty:
+                cur_img = None
+
+            if cur_img:
+                logger.debug("Processing image from motion detection notification...")
+                detected_results, detected_img_path = self._detect(cur_img["img"])
+                self.check_for_termination_requests()
+
+                if detected_results and detected_img_path:
+                    # Trigger image update in WADAS mainwindow
+                    self.update_image.emit(detected_img_path)
```

Comment:
No, we don't update mainwindow with classification detail if classified animal differs from the target. We can only log that the classified animal is different from the target. We can report in log the classified animal but no action is taken in mainwindow.